docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
+
+**DQL** is a SQL like query language for creating different views or calculations on your data. It
 supports:
 
-- Fetching pages associated with tags, folders, links, and so on.
-- Filtering pages / data by simple operations on fields, like comparison, existence checks, and so on.
-- Sorting results based on fields.
+- Choosing a **output format** of your output (the [Query Type](./query-types.md))
+- Fetch pages **from a certain [source](../reference/sources.md)**, i.e. a tag, folder or link
+- **Filtering pages/data** by simple operations on fields, like comparison, existence checks, and so on
+- **Transforming fields** for displaying, i.e. with calculations or splitting up multi-value fields
+- **Sorting** results based on fields
+- **Grouping** results based on fields
+- **Limiting** your result count
 
-The query language supports the following view types, described below:
+Let's have a look at how  we can put DQL to use.
 
-1. **TABLE**: The traditional view type; one row per data point, with several columns of field data.
-2. **LIST**: A list of pages which match the query. You can output a single associated value for each page.
-3. **TASK**: A list of tasks whose pages match the given query.
-4. **CALENDAR**: A calendar view displaying each hit via a dot on its referred date
+## General Format of a DQL Query
 
-Read more about the available types [here](./query-types.md)
+Every query follows the same structure and consists of
 
-## General Format
+- exactly one **Query Type** with, depending on your Query type, zero, one or many [fields](../annotation/add-metadata.md) for displaying
+- zero or one **FROM** data command with one to many [sources](../reference/sources.md)
+- zero to many other **data commands** with one to many [expressions](../reference/expressions.md) and/or other infos depending on the data command 
 
-The general format for queries is:
+Abstractly speaking, a query conforms the following pattern:
 
 ~~~
 ```dataview
-TABLE|LIST|TASK <field> [AS "Column Name"], <field>, ..., <field> 
+<QUERY-TYPE> <fields>
 FROM <source>
-WHERE <expression>
-SORT <expression> [ASC/DESC]
-... other data commands
+<DATA COMMAND> <expression>
 ```
 ~~~
 
-Only the [**Query Type**](./query-types.md) (table/list/task/calendar) statement is required - if the "from" statement is omitted, the query runs for all files in
-your vault. You can specify [data commands](./data-commands.md) like `WHERE` multiple times; they will run in the order they are written. 
+!!! hint "Only the Query Type is mandatory."
+
+The following sections will walk through the theory in more detail.
+
+## Choose a Output Format
+
+The final output appareance of a query is determined by its **Query Type**. There are four available:
+
+1. **TABLE**: The traditional view type; one row per result, with several columns of field data.
+2. **LIST**: A list of pages which match the query. You can output one field for each page alongside their file links.
```

Comment:
Similar to an earlier comment, let's be more precise about our terms since the word List is being overloaded. Consider using bullet list for this and for tasks.